### PR TITLE
add moar es6 things

### PR DESCRIPTION
**Pull request by shuvalov-anton**
Hey guys! Sorry if I'm annoying, but I'd added some es6 goodies to your awesome code :D

### CODE REVIEW

File: lib/posthtml.js [hunk 1]
```diff
@@ -5,65 +5,53 @@ var tmpl = new (require('bh').BH);
 class Posthtml {
 
     constructor(plugins) {
+        this.plugins = plugins || [];
+    }
```
Comment (iamstarkov) [inline lib/posthtml.js]: why not?
```js
constructor(plugins=[]) {
  this.plugins = plugins;
}
``

**Comment by verybigman** [comment]
Thanks! It's cool refactoring;) we have not time for super clean code:)

File: lib/posthtml.js (hunk 1 above)
Comment (shuvalov-anton) [inline lib/posthtml.js]: sure!

Comment (verybigman) [inline lib/posthtml.js]: :+1:

**Comment by verybigman** [comment]
@shuvalov-anton rebase?:)

**Comment by shuvalov-anton** [comment]
@verybigman squashed!

**Comment by verybigman** [comment]
Thanks:) you awesome!

**Comment by voischev** [comment]
Thanks! Cool! %)

**Pull request merged**